src/engine/strat_engine/backstore/metadata/mda.rs

```diff
@@ -367,12 +367,25 @@ impl MDARegions {
     }
 }
 
+#[derive(Debug, Default, PartialEq, PartialOrd)]
```

Comment:
Please derive the same set of properties as the other examples.

Reply:
That's what I did: I took the derive from `MDADataSize` which is basically the same. Or are you referring to another type?